flow/envs/green_wave_env.py

```diff
@@ -226,7 +226,8 @@ def _apply_rl_actions(self, rl_actions):
 
     def compute_reward(self, rl_actions, **kwargs):
         """See class definition."""
-        return rewards.penalize_tl_changes(rl_actions >= 0.5, gain=1.0)
+        # penalize traffic light changes for occurring
```

Comment:
Do you have any thoughts on how we can make this nicer with the threshold? It seems like this is really ripe for mistake i.e. if your env doesn't use 0.5 as its switching threshold you'll accidentally create a bug.